src/components/NegotiationList.vue

```diff
@@ -0,0 +1,217 @@
+            <li class="list-group-item list-group-item-action">Accepted <span class="badge badge-primary rounded-pill">0</span></li>
+        </ul>
+    </div>
+    <div v-if="showModal" class="modal">
+        <div class="modal-dialog">
+            <div class="modal-content">
+                <div class="modal-header">
+                    <h1 class="modal-title" id="exampleModalLabel">{{ negotiation.payload.project.title }}</h1>
+                </div>
+                <div class="modal-body">
+                    {{ negotiation.payload }}
+                    {{ negotiation.status }}
+                </div>
+                <div class="modal-body">
+                    <label for="actions">Respond:</label>
+                    <select v-model="selectedItem">
+                        <option v-for="response in response_options" :value="response">{{ response }}</option>
+                    </select>
+                    <p>Selected item: {{ selectedItem }}</p>
+                </div>
+                <div class="modal-footer">
+                    <button type="button" class="btn btn-secondary" @click="showModal = false">Close</button>
+                    <button type="button" class="btn btn-primary" @click="updateNegotiation">Submit</button>
+                </div>
+            </div>
+        </div>
+    </div>
+</template>
+
+<script>
+
+import {mapActions} from "vuex";
+
+export default {
+  name: "NegotiationsList",
+  props: {
+    negotiations: Array
+  },
+    data() {
+        return {
+            headers: [
+                "id",
+                "title",
+                "status"
+            ],
+            showModal: false,
+            negotiation: null,
+            response_options: [],
+            selectedItem: ''
+        }
+    },
+    methods: {
+        ...mapActions(["updateNegotiationStatus", "retrievePossibleEvents"]),
+        async updateNegotiation() {
+            await this.updateNegotiationStatus({
+                negotiationId: this.negotiation.id,
+                event: this.selectedItem
+            });
+            this.showModal = false
+        },
+        interactModal(negotiation){
+            this.showModal = true;
+            this.negotiation = negotiation;
+            this.loadPossibleEvents();
+
+        },
+        loadPossibleEvents(){
+            this.retrievePossibleEvents({
+                negotiationId: this.negotiation.id
+            }).then(data => {
+                this.response_options = data;
+            })
+        },
+      abandonRequest() {
+            if(confirm("Are you sure you want to abandon this request?")) {
+                console.log("deleting")
+            }
+        }
+    }
+}
+</script>
+
+<style scoped>
+
+.editButton {background-color: #008CBA; border: none} /* Blue */
+.abandonButton {background-color: #f44336;border: none} /* Red */
+.styled-table {
```

Comment:
To have a coherent style, it should be better to use bootstrap's table with table classes instead of defining a new table

https://getbootstrap.com/docs/5.0/content/tables/